df.history.xml

```diff
@@ -984,48 +1033,47 @@
         <int16_t name='caste' ref-target='caste_raw' aux-value='$$.race'/>
 
         <int32_t name='impersonated_hf' ref-target='historical_figure' comment="only when Impersonating"/>
-        <compound is-union='true' union-tag-attr='id_tag'>
-            <int32_t name='histfig_id' ref-target='historical_figure'/>
-            <int32_t name='nemesis_id' ref-target='nemesis_record'/>
-        </compound>
+        <int32_t name='originator_hf' ref-target='historical_figure'/>
```

Comment:
Toady's headers show no evidence of this ever storing a nemesis ID; however, one of our plugins is looking at "histfig_id" which I changed to "originator_hf" to reduce ambiguity.